shimmingtoolbox/shim/shim_utils.py

```diff
@@ -173,15 +173,36 @@ def phys_to_shim_cs(coefs, manufacturer):
         # Order 1
         if len(coefs) >= 3:
             # Change from RAS to LAI (ShimCS)
-            coefs[0] = -coefs[0]  # X
-            coefs[2] = -coefs[2]  # Z
+            if isinstance(coefs[0], np.ndarray):
+                if None not in coefs[0]:
+                    coefs[0] = -coefs[0]  # X
+            else:
+                if coefs[0] is not None:
+                    coefs[0] = -coefs[0]  # X
+
+            if isinstance(coefs[2], np.ndarray):
+                if None not in coefs[2]:
+                    coefs[2] = -coefs[2]  # Z
+            else:
+                if coefs[2] is not None:
+                    coefs[2] = -coefs[2]  # X
```

Comment:
```suggestion
                    coefs[2] = -coefs[2]  # Z
```

Reply:
The merge with master fixed the typo: [91a44d7](https://github.com/shimming-toolbox/shimming-toolbox/pull/483/commits/91a44d772c8a5f09a0875dcbff6cbda865be105c)